src/app/components/mnoe-api/teams.svc.coffee

```diff
@@ -69,6 +69,8 @@ angular.module 'mnoEnterpriseAngular'
       MnoeApiSvc.one('teams', teamId).customPUT(payload, '/remove_users').then(
         (response) ->
           response = response.plain()
+          # Update the team's users in the frontend
+          _.find(_self.teams, (team) -> team.id == teamId).users = response.team.users
```

Comment:
You can use the the `_.matchesProperty` iteratee shorthand.
```coffeescript
_.find(_self.teams, ['id' ,teamId])
```

Or the `_.matches` iteratee shorthand to be consistent with `@deleteTeam`
```coffeescript
_.find(_self.teams, {id: teamId})
```

Reply:
Good, changing in @addUsers as well